style.css

```diff
@@ -754,4 +678,116 @@ button:enabled {
 button:hover {
   background: #fff5e1;
   border: 1px solid #f55800;
```

Comment:
- [OPTIONAL] kindly add `color: #f55800` property to change the text color to `#f55800` when hovering on the button 👍